src/ui/public/inspector/adapters/request/request_responder.ts

```diff
@@ -43,6 +43,13 @@ export class RequestResponder {
       ...(this.request.stats || {}),
       ...stats,
     };
+    if (this.request.startTime) {
```

Comment:
We will always set `startTime` and it's a mandatory field on the interface, so no need for the `if` here.